testing/src/main/java/io/stargate/it/http/graphql/graphqlfirst/BulkInsertValidationTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.it.http.graphql.graphqlfirst;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import io.stargate.it.driver.CqlSessionExtension;
+import io.stargate.it.driver.TestKeyspace;
+import io.stargate.it.http.RestUtils;
+import io.stargate.it.storage.StargateConnectionInfo;
+import java.util.Map;
+import org.assertj.core.api.AssertionsForClassTypes;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(CqlSessionExtension.class)
+public class BulkInsertValidationTest extends GraphqlFirstTestBase {
+
+  private static GraphqlFirstClient CLIENT;
+  private static String KEYSPACE;
+
+  @BeforeAll
+  public static void setup(StargateConnectionInfo cluster, @TestKeyspace CqlIdentifier keyspaceId) {
+    CLIENT =
+        new GraphqlFirstClient(
+            cluster.seedAddress(), RestUtils.getAuthToken(cluster.seedAddress()));
+    KEYSPACE = keyspaceId.asInternal();
+  }
+
+  @Test
+  @DisplayName("Should fail when deploying schema with a bulk insert returning non-list.")
+  public void shouldFailToDeploySchemaWithABulkInsertReturningNonList() {
+    // given, when
+    Map<String, Object> errors =
+        CLIENT
+            .getDeploySchemaErrors(
+                KEYSPACE,
+                null,
+                "type User @cql_input {\n"
+                    + "  id: ID!\n"
+                    + "  name: String\n"
+                    + "  username: String\n"
+                    + "}\n"
+                    + "type Query { user(id: ID!): User }\n"
+                    + "type InsertUserResponse @cql_payload {\n"
+                    + "  applied: Boolean!\n"
+                    + "  user: User!\n"
+                    + "}\n"
+                    + "type Mutation {\n"
+                    + "  bulkInsertUsers(users: [UserInput!]): User\n"
+                    + "}")
+            .get(0);
+
+    // then
+    AssertionsForClassTypes.assertThat(getMappingErrors(errors))
```

Comment:
Copy/paste issue again.
I've looked for a way to pick `Assertions` automatically in IDEA, but I can't find anything...